computer-vision/README.md

```diff
@@ -0,0 +1,42 @@
+# Computer Vision
+
+The repository shall focus on the traditional aspects of computer vision, particularly multiview geometry. 
+
+## Pre - Requisites
+1. It is expected that one is familiar with some of the concepts of Linear Algebra. No course can beat the one provided by [Gilbert Strang](https://www.youtube.com/playlist?list=PLE7DDD91010BC51F8). Make sure that one sticks patiently as some of the concepts might seem daunting initially. For gaining some motivation and intuition for some of the concepts, one can refer to the [Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab).
+2. For implementation of some of the concepts, [OpenCV](https://opencv.org/releases/) will help you. Note that you are required to have some background in python. There are many tutorials available, such as [the one by Corey Schafer](https://www.youtube.com/watch?v=YYXdXT2l-Gg&list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU), or [by sentdex](https://www.youtube.com/playlist?list=PLQVvvaa0QuDe8XSftW-RAxdo6OmaeL85M). 
```

Comment:
Python isn't an obligation, one can implement in C++ too. Such notes are not required. Keep some stuff Self-Explanatory.  Put OpenCV in Libraries for Implementation.